src/PolarSSL++/SslContext.h

```diff
@@ -127,13 +127,13 @@ class cSslContext abstract
 	/** The callback used by PolarSSL when it wants to read encrypted data. */
 	static int ReceiveEncrypted(void * a_This, unsigned char * a_Buffer, size_t a_NumBytes)
 	{
-		return ((cSslContext *)a_This)->ReceiveEncrypted(a_Buffer, a_NumBytes);
+		return (static_cast<cSslContext *>(a_This))->ReceiveEncrypted(a_Buffer, a_NumBytes);
```

Comment:
No, because its a void pointer. You can do a reinterpret cast by casting to void and back, but I think that's UB.